app/views/admin/health/patient_referrals/_patient_referral.haml

```diff
@@ -1,5 +1,5 @@
 .c-card.mb-2.c-card-overflow-inherit
-  .c-card__content.jPatientReferral
+  .c-card__content.jPatientReferral{class: [pr.assigned? && :jPatientReferralAssigned]}
```

Comment:
what does this do?  Usually [] would look for an array, so is this `[true]` if `pr.assigned?`?  `&& :jPatientReferralAssigned` will always return true I think.  Class then gets converted to a string, so I think this would be the equivalent of class: "#{pr.assigned? && :jPatientReferralAssigned}" (which isn't as nice, but I'm confused by && with a symbol.)

Reply:
This conditionally assigns the class based on the result of `pr.assigned?`. I found it from here https://stackoverflow.com/a/30763508

How do you typically do conditional classes in haml? Another option is `{ class: ("jPatientReferralAssigned" if pr.assigned?) }` 

I threw in this classname only because I needed a way to identify whether a patient was already assigned or not [here](https://github.com/greenriver/hmis-warehouse/pull/2165/files#diff-9e67fba38785b2feb5ccb21089a6426f985b6fbf4afaf382a30492f0fe62dbde). Class name is not my favorite, I can change this to store it in the HTML as a boolean data attribute like `data-is-assigned` instead.

Reply:
I would tend to do something like 
```
referral_assigned_class = if pr.assigned? then : jPatientReferralAssigned else '' end
{ class: referral_assigned_class }
```

Reply:
okay. I think it's nice that you don't necessarily need to pull it out into a variable though. I changed this from a class to a data attribute though, so no longer relevant. Now it's

```
.c-card__content.jPatientReferral{ data: { is_assigned: pr.assigned? }}
```